go.mod

```diff
@@ -4,7 +4,7 @@ go 1.21
 
 require (
 	aead.dev/minisign v0.2.0
-	github.com/bufbuild/buf v1.27.2
+	github.com/bufbuild/buf v1.27.3-0.20231106140608-d7a87496b879
```

Comment:
We'll also need to update the .github/workflow files for CI to build with the newer buf version (@mfridman just had to do this for another new field added to buf.plugin.yaml).

Reply:
makes sense - looks like the revert was this PR: https://github.com/bufbuild/plugins/pull/886, so when the downstream PR is merged in buf I can update these back to pulling from `main` (assuming we don't get out a release in the meantime)

Reply:
This should be released in https://github.com/bufbuild/buf/releases/tag/v1.28.0 - I'm assuming we'll get that pulled in by Dependabot next week?